tests/test_commands.py

```diff
@@ -89,8 +89,11 @@ def test_clearindex(self):
             self.assertRegexpMatches(result, regex)
 
     def test_clearindex_with_args(self):
-        call_command('algolia_clearindex', stdout=self.out,
-                     model=['Website'], batchsize=3)
```

Comment:
why removing batchsize? :thinking:

Reply:
The `clearindex` command do not take a `batchsize` argument ;)
(The extra arguments are now checked in Django > 2.0, so the test fail on this version, for a good reason)